src/client/components/MessageCenter/Topic/Footer/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useParams } from 'react-router-dom'
+
+import { Objects } from 'utils/objects'
+
+import { MessageTopic, MessageTopicType } from 'meta/messageCenter'
+
+import { useAppDispatch } from 'client/store'
+import { useAssessment, useCycle } from 'client/store/assessment'
+import { MessageCenterActions } from 'client/store/ui/messageCenter'
+import { useCountryIso } from 'client/hooks'
+import { DataCell, DataGrid } from 'client/components/DataGrid'
+import TextArea from 'client/components/Inputs/TextArea'
+
+import { useShowActions } from './hooks/useShowActions'
+
+const Footer: React.FC<{ topic: MessageTopic }> = ({ topic }) => {
```

Comment:
Extracted the Footer in it's own component

Is `React.FC<Props>` preferred when passing single prop?